network-api/networkapi/wagtailpages/templates/fragments/product_criterion_primary_info.html

```diff
@@ -1,42 +1,47 @@
-{% load i18n static %}
+{% load i18n static wagtailcore_tags %}
 
 <div class="primary-info">
-  <h3 class="tw-mb-4 tw-font-zilla tw-text-2xl tw-leading-7 tw-flex tw-items-center">
-    {{ label }}
-    {% if info != None and info != "" %}
-      <a href="{{info}}">
-        <img src="{% static "_images/buyers-guide/icon-info-circle.svg" %}" alt="{% trans "information" context "icon description for screen readers" %}" class="tw-h-4 tw-w-4 tw-ml-2 tw-mb-1">
-      </a>
-    {% endif %}
-  </h3>
-
-  <p class="rating pni-product-smaller-body mb-0">
-    {% if value == "CD" %}
-      {% if show_value_as_symbol %}
-      <img src="{% static "_images/buyers-guide/score/icon-unknown.svg" %}" alt="{% trans "Can’t determine" context "Unknown rating" %}">
-      {% else %}
-      {% trans "Can’t determine" context "Unknown rating" %}
-      {% endif %}
-    {% elif value == "NA" %}
-      {% trans "Not applicable" %}
-    {% elif value == "Yes" %}
-      {% if show_value_as_symbol %}
-      <img src="{% static "_images/buyers-guide/score/icon-yes.svg" %}" alt="{% trans "Yes" %}">
-      {% else %}
-      {% trans "Yes" %}
-      {% endif %}
-    {% elif value == "No" %}
-      {% if show_value_as_symbol %}
-      <img src="{% static "_images/buyers-guide/score/icon-no.svg" %}" alt="{% trans "No" %}">
-      {% else %}
-      {% trans "No" %}
+    <h3 class="tw-mb-4 tw-font-zilla tw-text-2xl tw-leading-7 tw-flex tw-items-center">
+      {{ label }}
+      {% if info != None and info != "" %}
+        <a href="{{info}}">
+          <img src="{% static "_images/buyers-guide/icon-info-circle.svg" %}" alt="{% trans "information" context "icon description for screen readers" %}" class="tw-h-4 tw-w-4 tw-ml-2 tw-mb-1">
+        </a>
       {% endif %}
+    </h3>
+    {% if rich_text_value %}
+      <div class="criterion-richtext-value mb-0">
+        {{ value|richtext }}
+      </div>
     {% else %}
-      {% if show_value_as_symbol %}
-      <img src="{% get_static_prefix %}_images/buyers-guide/score/icon-{{value|lower}}.svg" alt="{{value}}">
+    <p class="rating pni-product-smaller-body mb-0">
+      {% if value == "Can’t Determine" or value == "CD"%}
```

Comment:
Why do we need the dual check here? This should never be the full expanded string, so if it is, that's... a problem.